src/librustdoc/html/markdown.rs

```diff
@@ -308,7 +308,8 @@ impl<'a, I: Iterator<Item = Event<'a>>> Iterator for CodeBlocks<'_, 'a, I> {
                 builder = builder.crate_name(krate);
             }
             let doctest = builder.build(None);
-            let (test, _) = doctest.generate_unique_doctest(&test, false, &opts, krate);
+            let (wrapper, _) = doctest.generate_unique_doctest(&test, false, &opts, krate);
+            let test = wrapper.to_string();
```

Comment:
```suggestion
            let (wrapped, _) = doctest.generate_unique_doctest(&test, false, &opts, krate);
            let test = wrapped.to_string();
```

Nit: it's not just a wrapper, but the wrapper in combination with the inner code that has been wrapped.  we make this distinction in the JSON output, so maybe we should stick to that terminology here.